components/Feed.vue

```diff
@@ -29,6 +29,11 @@ export default {
       }
       return false
     },
+    getFeedFilePathWithBase()
```

Comment:
Thanks for your contribution!

There's already been a computed function getting the feed path. Please whether remove it or modify it.
Besides, you can wrap the path with `$withBase`. Ref: https://vuepress.vuejs.org/guide/assets.html#base-url

Reply:
Thanks for the information! I modified it.